### PR TITLE
Added parser raw_data argument documentation

### PR DESCRIPTION
**Pull request by domachad**

### CODE REVIEW

**Review by dwapstra — APPROVED** [review]

File: docs/cookbooks/parsers.rst [hunk 1]
```diff
@@ -255,8 +255,32 @@ This will execute all the commands without raising an exception. Any silenced
...
+
+.. code-block:: python
+
+    dev = testbed.devices['R1_xe']
+    out = dev.parse('show version', raw_data=True)
+    out.raw_output
+    [{'command': 'show version', 'kwargs': {}, 'output': 'Cisco IOS XE Software, Version 17.03.01a\r\n...'}]
+    ...
+
+The device has to be connected, otherwise the attribute will not be created and you will only get the parsed output.
```
Comment (tahigash) [inline docs/cookbooks/parsers.rst:276]: not work with `output=output`?

File: docs/cookbooks/parsers.rst [hunk 2]
```diff
@@ -255,8 +255,32 @@ This will execute all the commands without raising an exception. Any silenced
...
+
+.. note::
+
+   You need genie 22.5 or greater to use this feature.
```
Comment (tahigash) [inline docs/cookbooks/parsers.rst:280]: version should be same among pyATS packages. so let's say as pyATS
```suggestion
   You need pyATS 22.5 or greater to use this feature.
```

**Review by tahigash — CHANGES_REQUESTED** [review]
Please write something in PR description even with small note.

File: docs/cookbooks/parsers.rst (hunk 1 above)
Comment (domachad) [reply docs/cookbooks/parsers.rst:276]: Currently no. Should it? 
We collect the raw output by decorating execute, which is not called when the device is disconnected.
We can reproduce the same behavior for disconnected devices

Comment (domachad) [reply docs/cookbooks/parsers.rst]: @tahigash The code was modified to work with disconnected devices and I removed this line from the documentation.
Can you please re-review both PRs?
https://wwwin-github.cisco.com/pyATS/genie/pull/222

**Review by tahigash — APPROVED** [review]

**Pull request merged**